android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -80,11 +80,10 @@ function setOskHeight(h) {
   if(h > 0) {
```

Comment:
`init`, `setBannerHeight` above still reference `com.keyman.singleton`, need cleanup to `keyman.`.

Line 13 references `com.keyman.KeymanSentryManager`. We're trying to eliminate use of this `com.keyman.` namespacing over time, so it'd be good to fix this but I don't think KMW currently exports `KeymanSentryManager` in any other way; is that correct @jahorton ?

Reply:
Good catch.

I'm also not sure if there's a way to use `keyman` in l.45
```
com.keyman.osk.Banner.DEFAULT_HEIGHT =
    Math.ceil(window.jsInterface.getDefaultBannerHeight() / window.devicePixelRatio);
```

Reply:
Pretty sure that's the only way it's currently exposed, yeah.

I don't think it's worth large amounts of effort to avoid `com.keyman` right now - when we shift to ES modules & `esbuild` use, that'll make so much of this so much easier.  Even if that shift itself will be a significant amount of work.